.github/workflows/check-pod.yml

```diff
@@ -12,15 +12,23 @@ jobs:
 
       # Run the steps we document in the Release Process.
       # unzip commands included as proof-of-life for the Carthage output.
-      - run: |
-          ruby --version
-          echo -n "carthage version: " && carthage version
-          echo -n "pod version: " && pod --version --verbose
-          make --version
-          make update
-          make carthage_package
+      - name: Print Ruby version
```

Comment:
This is perhaps a bit OTT to have as multiple steps. I would have perhaps preferred a single step `name`d "Print Tooling Versions" or similar. This'll look a bit busy in the run UI.

Reply:
As agreed on Slack, we’ll merge to get CO fixed, and I’ll address this later.